src/plat/tqma8xqp1gb/config.cmake

```diff
@@ -12,6 +12,7 @@ if(KernelPlatformTqma8xqp1gb)
     declare_seL4_arch(aarch64)
     set(KernelArmCortexA35 ON)
     set(KernelArchArmV8a ON)
+    set(KernelArmGicV3 ON)
```

Comment:
We should remove this as it's from before the config was removed.

Reply:
Agreed. That is a left over. I'll wait until @lsf37 is able to say something about the proof impacts before I make any additional changes.